tests/core/test_expectation_suite.py

```diff
@@ -93,14 +93,6 @@ def suite_with_single_expectation(
 class TestInit:
     """Tests related to ExpectationSuite.__init__()"""
 
-    @pytest.mark.unit
```

Comment:
Why remove this test?

Reply:
we no longer grab the store in the `__init__` so we won't hit this error - this is a change in behavior but it more in line with how the rest of the domain objects work (you'll hit the data context dependency error when trying to use the object rather than at instantiation time)